controls/datagrid/filtering/programmatic-filtering.md

```diff
@@ -217,4 +223,4 @@ Add the `DelegateFilterDescriptor` to the `RadDataGrid` instance:
 dataGrid.FilterDescriptors.Add(new DelegateFilterDescriptor() { Filter = new CustomFilter()});
 ```
 
->important You could check our Programmatic Filtering example referring to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}).
\ No newline at end of file
+>important You could check our Programmatic Filtering example referring to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}).
```

Comment:
When referring to the SDKBrowser, always use the same sentence as a template. This makes the pages predictable and saves the developer time. You can adjust the sentence below so it fits this specific scenario:

"For a runnable example with the Button Getting Started scenario, see the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and go to Button > Getting Started."

Reply:
Updated